### DISCUSSION TITLE
蓝灯不好用，申请退款

### DISCUSSION THREAD

**Discussion by Lewis0000221** [getlantern/lantern-forum-cn#981] [Q&A]
别人套餐比你便宜，网速比你快，节点比你稳定，有固定节点，不需要等节点稳定才能上网，基本上不掉线，蓝灯用着用着会掉线，网速也不稳定，等你节点稳定才能上网，我用其他VPN已经连上十多分钟了，离谱不？？？蓝灯体验感超差！！！！我已经换VPN了，请退款！！！
![Uploading 微信截图_20241022102044.png…]()

**Comment by MaxMKMax** [comment]
@Lewis0000221 您好. 请您提供我支付单据和账号邮箱. 把支付单据发到我的邮箱吧. 然后这里告述我您使用哪一个邮箱发邮件给我的.
mike@getlantern.org

**Comment by Raymondlee321** [comment]
***@***.***
支付单据已通过附件上传

Mike K ***@***.***> 于2024年10月24日周四 00:25写道：
